README.md

```diff
@@ -133,19 +133,20 @@ The data table can be created with the following `options`:
   - `template`
   - `width`: Width, in pixels (default: `150`)
 - `emptyMessage`: Message to show when array is presented but contains no values (default: `'No data to display'`)
-- `footerHeight`: Footer height in pixels. Pass a  `falsey` value for no footer (default: `0`)
+- `footerHeight`: Footer height in pixels. Pass a  `falsey` value for no footer. (default: `0`)
 - `headerHeight`: Header height in pixels. Pass a  `falsey` value for no header (default: `30`)
 - `loadingMessage`: Loading message presented when the array is undefined (default: `'Loading...'`)
 - `multiSelect`: Whether users can select multiple items (default: `false`)
 - `paging`
-  - `externalPaging`. Default `false`
-  - `size`. Default `undefined`
-  - `count`. Default `0`
-  - `offset`. Default `0`
-  - `loadingIndicator`. Default `false`
+  - `externalPaging`. (default: `false`)
+  - `internal`. (default `false`)
```

Comment:
Went with `internal` rather than `internalPaging` because we're already in a `paging` object. Let's refactor `externalPaging`  to match later...